RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## v1.6.0
+* dependency updates
+* lower TTL for containers
```

Comment:
```suggestion
* lower the TTL to 60s for container names
```

maybe a bit more context would be helpful

Reply:
you have to squash this commit into your release notes commit, right now it comes after the release commit so it will not be included in the tag.